src/view/LoginView.java

```diff
@@ -1,4 +1,128 @@
 package view;
 
-public class LoginView {
+import interface_adapter.login.LoginController;
+import interface_adapter.login.LoginState;
+import interface_adapter.login.LoginViewModel;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+public class LoginView extends JPanel implements ActionListener, PropertyChangeListener {
+
+    public final String viewName = "Login";
+
+    private final LoginViewModel loginViewModel;
+    private final LoginController loginController;
+
+    private final JTextField usernameInputField = new JTextField(15);
+    private final JLabel usernameErrorField = new JLabel();
+    private final JPasswordField passwordInputField = new JPasswordField(15);
+    private final JLabel passwordErrorField = new JLabel();
+
+
+
+    final JButton logIn;
+    final JButton createNewAccount;
+
+    public LoginView(LoginViewModel loginViewModel, LoginController controller) {
+
+        this.loginViewModel = loginViewModel;
+        this.loginController = controller;
+        this.loginViewModel.addPropertyChangeListener(this);
+
+        JLabel title = new JLabel("Login Screen");
+        title.setAlignmentX(Component.CENTER_ALIGNMENT);
+
+        LabelTextPanel usernameInfo = new LabelTextPanel(
+                new JLabel("Username"), usernameInputField);
+        LabelTextPanel passwordInfo = new LabelTextPanel(
+                new JLabel("Password"), passwordInputField);
+
+        // add buttons for log_in and sign_up.
+        JPanel buttons = new JPanel();
+
+        logIn = new JButton(loginViewModel.LOGIN_BUTTON_LABEL);
+        buttons.add(logIn);
+        createNewAccount = new JButton(loginViewModel.CREATE_BUTTON_LABEL);
+        buttons.add(createNewAccount);
+
+        logIn.addActionListener(
+                new ActionListener() {
+                    public void actionPerformed(ActionEvent evt) {
+                        if (evt.getSource().equals(logIn)) {
+                            LoginState currentState = loginViewModel.getState();
+
+                            loginController.execute(
+                                    currentState.getUsername(),
+                                    currentState.getPassword()
+                            );
+                        }
+                    }
+                }
+        );
+
+        usernameInputField.addKeyListener(new KeyListener() {
```

Comment:
Utilizing KeyListener for usernameInputField and passwordInputField might not be the best choice, as it cannot capture all types of user input (like pasting text). You might want to consider using DocumentListener instead.